docs/getting-started/quick-start.md

```diff
@@ -25,7 +25,7 @@ npm install ckeditor5
 
 Now you can import all the modules from the `ckeditor5` package. Additionally, you have to import CSS styles separately.
 
-**Importing and registering UI translations is optional for the English language.** For any other language, use imported translations, as in the {@link getting-started/setup/configuration setup section}.
+**Importing and registering UI translations is optional for American English.** To use the editor in any other language, use imported translations, as shown in the {@link getting-started/setup/configuration setup section}.
```

Comment:
```suggestion
**Importing and registering UI translations is optional for English.** To use the editor in any other language, use imported translations, as shown in the {@link getting-started/setup/configuration setup section}.
```

Reply:
I insist. We do not have separate translations, do we?

Reply:
It seems that we have:

> As far as I can see, we have en (default), en-au and en-gb. That's why I suggested explicitly saying that American English is the default.

I also asked @pszczesniak, and he confirmed.

Reply:
Why don't we have this documented and only list English in out langauge guide? o\_O This is a pretty meaningful rewrite.  
  
OK\< Keep American English, then. I'll split the issue on that.

Reply:
I looked through the language guides, and you're right - there is only "English" without dialects.

Reply:
We actually do have community translations for these two bad boys: \`au\` and `gb` https://github.com/ckeditor/ckeditor5/tree/master/packages/ckeditor5-core/lang/translations: